src/adr/0041-james-servers-offering.md

```diff
@@ -0,0 +1,159 @@
+# 41. James server offering
+
+Date: 2020-06-22
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+The Apache James project is willing to provide ready-to-use Mail Servers for our users.
+
+There exists several use cases for such servers:
+
+ - **Self hosting**: a hobbyist, or a small organization is willing to host a mail service. They 
+have limited infrastructure, time and knowledge and are mostly looking for an easy to use 
+solution.
```

Comment:
Again for consistency, maybe consider these changes?

is willing to host --> desires to operate ("host" is not really defined anywhere)
easy to use --> easy-to-operate